doxygen/mbedtls.doxyfile

```diff
@@ -1594,7 +1594,7 @@ SEARCH_INCLUDES        = YES
 # contain include files that are not input files but should be processed by
 # the preprocessor.
 
-INCLUDE_PATH           =
+INCLUDE_PATH           = ../include
```

Comment:
Just a check. There are include files in the ./library directory as well ("internal" includes though). Do we need to include this path here as well?

Reply:
I assume that these are internal headers, and we don't want to document them.